src/core/viz/expressions/Animation.js

```diff
@@ -91,47 +23,47 @@ export class Fade extends BaseExpression {
  * const viz = new carto.Viz(`
  *   width: 2
  *   color: ramp(linear(clusterAvg($temp), 0,30), tealrose)
- *   filter: torque($day, 40, fade(0.1, 0.3))
+ *   filter: animation($day, 40, fade(0.1, 0.3))
  * `);
  *
  * @example <caption>Temporal map by $date (of date type), with a duration of 40 seconds, fadeIn of 0.1 seconds and fadeOut of 0.3 seconds. (String)</caption>
  * const viz = new carto.Viz(`
  *   width: 2
  *   color: ramp(linear(clusterAvg($temp), 0,30), tealrose)
- *   filter: torque(linear($date, time('2022-03-09T00:00:00Z'), time('2033-08-12T00:00:00Z')), 40, fade(0.1, 0.3))
+ *   filter: animation(linear($date, time('2022-03-09T00:00:00Z'), time('2033-08-12T00:00:00Z')), 40, fade(0.1, 0.3))
  * `);
  *
- * @example <caption>Using the `getSimTime` method to get the simulated time.</caption>
+ * @example <caption>Using the `getTimestamp` method to get the animation current timestamp</caption>
```

Comment:
We shouldn't name it timestamp since it will be a number when the input is a number